style.css

```
@@ -1,16 +1,85 @@
-#number-guess,
-#result,
-#history {
-  max-width: 400px;
-  margin: 0 auto;
+body {
+    margin: auto;
+    background-image: url('images/image-asset.jpeg');
+    background-size: cover;
```

- [ ] Change background from cover to fill.

```suggestion
body {
    margin: auto;
    background-image: url('images/image-asset.jpeg');
    background-size: fill;
```
or 
```suggestion
body {
    margin: auto;
    background-image: url('images/image-asset.jpeg');
    background-size: 100vw 700px;
```